<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuscarTarea</class>
 <widget class="QMainWindow" name="BuscarTarea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>380</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_22">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Filtrar:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtfilter"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>Buscar</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/rescr/search_find</normalon>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Materia:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboMat"/>
          </item>
          <item>
           <widget class="QCheckBox" name="chbMat">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>!100:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbNot100">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Grupo:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbGroup">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Pririodad:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spbPrio"/>
          </item>
          <item>
           <widget class="QCheckBox" name="chbPrio">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Entrega:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dtEnt">
            <property name="displayFormat">
             <string notr="true">dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbEnt">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Adelantar:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dtAd">
            <property name="displayFormat">
             <string notr="true">dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbAd">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tblRes">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragOnly</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::TargetMoveAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Progreso: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pbarProgress">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPrPr">
        <property name="text">
         <string>0%</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="txtDescrp">
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFix_Screen"/>
   <addaction name="actionH_img"/>
  </widget>
  <action name="actionFix_Screen">
   <property name="icon">
    <iconset resource="rsour.qrc">
     <normaloff>:/rescr/UpdateIcon</normaloff>:/rescr/UpdateIcon</iconset>
   </property>
   <property name="text">
    <string>Fix Screen</string>
   </property>
  </action>
  <action name="actionH_img">
   <property name="icon">
    <iconset resource="rsour.qrc">
     <normaloff>:/rescr/mov_obj</normaloff>:/rescr/mov_obj</iconset>
   </property>
   <property name="text">
    <string>H_img</string>
   </property>
   <property name="toolTip">
    <string>View Horary</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rsour.qrc"/>
 </resources>
 <connections/>
</ui>
